src/main/java/com/group2/catanbackend/gamelogic/GameLogicController.java

```diff
@@ -176,49 +176,46 @@ private boolean stealResource(Player playerToStealFrom, Player playerToGiveTo) {
         }
         return false;
     }
-    //TODO: error messages @daniel
-    private void makeAcceptMove(AcceptMoveDto acceptMove, Player player){
+    private void makeAcceptTradeOfferMove(AcceptTradeOfferMoveDto acceptMove, Player player){
         if(isSetupPhase)
-            throw new NotActivePlayerException(ErrorCode.ERROR_IS_SETUP_PHASE);
+            throw new InvalidGameMoveException(ErrorCode.ERROR_IS_SETUP_PHASE);
         if(currentTrade==null)//trade is gone
             throw new InvalidGameMoveException(ErrorCode.ERROR_TRADE_NOT_AVAILABLE);
-        if(activePlayer.getInGameID()!=acceptMove.getTradeOfferDto().getFromPlayer().getInGameID())
-            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES.formatted("active player"));
         if(!currentTrade.sameAs(acceptMove.getTradeOfferDto()))
-            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES.formatted("current trade not accepting trade"));//TradeOfferDtos are not the same
+            throw new InvalidGameMoveException(ErrorCode.ERROR_WRONG_TRADE);
         if (!player.resourcesSufficient(acceptMove.getTradeOfferDto().getGiveResources()))
-            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES.formatted("the REAL resources sufficient"));
+            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES);
         if(!activePlayer.resourcesSufficient(negateAllValues(acceptMove.getTradeOfferDto().getGetResources())))
-            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES.formatted("receiving player resources"));//trader does not have enough resources
-        computeAcceptMove(acceptMove.getTradeOfferDto(), player);
+            throw new InvalidGameMoveException(ErrorCode.ERROR_NOT_ENOUGH_RESOURCES);
+        computeAcceptTradeOfferMove(acceptMove.getTradeOfferDto(), player);
     }
-    private void computeAcceptMove(TradeOfferDto tradeOffer, Player player){//more
+    private void computeAcceptTradeOfferMove(TradeOfferDto tradeOffer, Player player){//more
         activePlayer.adjustResources(negateAllValues(tradeOffer.getGetResources()));
         activePlayer.adjustResources(negateAllValues(tradeOffer.getGiveResources()));
         player.adjustResources(tradeOffer.getGiveResources());
         player.adjustResources(tradeOffer.getGetResources());
         this.currentTrade=null;
         sendCurrentGameStateToPlayers();
-        //send new GameProgressDto? but with what content?
+        //optional: notify all users about happened trade
```

Comment:
why leave this comment in? this is gonna be the last commit in our project, either do it or delete the comment